test/python/compiler/test_transpiler.py

```diff
@@ -2393,23 +2395,6 @@ def test_qasm3_output(self, optimization_level):
         # itself doesn't throw an error, though.
         self.assertIsInstance(qasm3.dumps(transpiled).strip(), str)
 
-    @data(0, 1, 2, 3)
-    def test_qasm3_output_v1(self, optimization_level):
-        """Test that the output of a transpiled circuit can be dumped into OpenQASM 3 (backend V1)."""
-        with self.assertWarns(DeprecationWarning):
-            backend = Fake20QV1()
-
-            transpiled = transpile(
-                self._regular_circuit(),
-                backend=backend,
-                optimization_level=optimization_level,
-                seed_transpiler=2022_10_17,
-            )
-        # TODO: There's not a huge amount we can sensibly test for the output here until we can
-        # round-trip the OpenQASM 3 back into a Terra circuit.  Mostly we're concerned that the dump
-        # itself doesn't throw an error, though.
-        self.assertIsInstance(qasm3.dumps(transpiled).strip(), str)
-
```

Comment:
Should this have been deleted?

Reply:
I was jumping the gun a bit too much on this one since `BackendV1` is deprecated and being removed I opted to just remove this test. `BackendV1` doesn't support control flow instruction (as it can't represent them as allowed) and `_regular_circuit()`  adds an if test. So I opted to just remove this test because we cover the qasm and transpiler pieces in `test_qasm3_output` and it'll be removed anyway in the `BackendV1` test.

Reply:
Ah, I'd not realised that `_regular_circuit` included control flow.  Yeah, I can live with just removing this, even though it's marginally out of order.